tests/clients/VKTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace yiiunit\extensions\authclient\clients;
+
+use yiiunit\extensions\authclient\TestCase;
+
+/**
+ * @group VKontakte
+ */
+class VKTest extends TestCase
+{
+    public function testResponse()
+    {
+        $response = [
+            'error' => [
+                'error_code' => 5,
+                'error_msg' => '',
+                'request_params' => '',
+            ],
+            'response' => null,
+        ];
+        $attributes = [];
+        if (!empty($response['response'])) {
+            $attributes = array_shift($response['response']);
+        }
+
+        $this->assertEmpty($attributes);
+    }
```

Comment:
What kind of unit test is this? It is completely useless since it does not cover any internal code.
It should be either rewritten or removed.